packages/gatsby/src/query/pages-writer.js

```diff
@@ -119,6 +119,7 @@ const preferDefault = m => m && m.default || m
     writeAndMove(`pages.json`, JSON.stringify(pagesData, null, 4)),
     writeAndMove(`sync-requires.js`, syncRequires),
     writeAndMove(`async-requires.js`, asyncRequires),
+    writeAndMove(`match-paths.json`, JSON.stringify(matchPaths, null, 4)),
```

Comment:
we probably want to merge https://github.com/gatsbyjs/gatsby/pull/13012 first as these changes are done there as well.

Reply:
Nope, you found a mistake. There's no place for `match-paths.json` in this PR. Removed in 14a7af6